bookkeeper-server/src/test/java/org/apache/bookkeeper/client/BookieWriteLedgerTest.java

```diff
@@ -1,31 +1,31 @@
 /**
- *
  * Licensed to the Apache Software Foundation (ASF) under one
  * or more contributor license agreements.  See the NOTICE file
  * distributed with this work for additional information
  * regarding copyright ownership.  The ASF licenses this file
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
```

Comment:
the changes of line 9-12 are not needed here.

Reply:
done.